pkg/cli/image_cmd_test.go

```diff
@@ -1029,9 +1037,9 @@ func TestServerResponseGQL(t *testing.T) {
 			// IMAGE NAME    TAG       DIGEST    SIZE
 			// repo7         test:2.0  a0ca253b  15B
 			// repo7         test:1.0  a0ca253b  15B
-			So(actual, ShouldContainSubstring, "IMAGE NAME TAG DIGEST SIGNED SIZE")
-			So(actual, ShouldContainSubstring, "repo7 test:2.0 883fc0c5 false 492B")
-			So(actual, ShouldContainSubstring, "repo7 test:1.0 883fc0c5 false 492B")
+			So(actual, ShouldContainSubstring, "IMAGE NAME TAG DIGEST OS/ARCH SIGNED SIZE")
+			So(actual, ShouldContainSubstring, "repo7 test:2.0 883fc0c5 / false 492B")
```

Comment:
Why in some cases the column is `/`?

Reply:
In this case there is no OS or ARCH found in the config of the manifest. Should there me another character to represent no os/arch?